docs/development/testguide.rst

```diff
@@ -25,6 +25,105 @@ accessed through the ``python setup.py test`` command.
     The ``pytest`` project was formerly called ``py.test``, and you may
     see the two spellings used interchangeably in the documentation.
 
+.. _testing-dependencies:
+
+Testing Dependencies
+********************
+
+As of Astropy 3.0, the dependencies used by the Astropy test runner are
+provided by a separate package called ``pytest-astropy``. This package provides
+the ``pytest`` dependency itself, in addition to several ``pytest`` plugins
+that are used by Astropy, and will also be of general use to other packages.
```

Comment:
I would say managed rather than used, but if the plan is used, then I suggest to add pytest-mpl in there, too as astropy uses it.

Reply:
Hmm, that's a good thought. Does @astrofrog think that `pytest-mpl` should be added to `pytest-astropy` as well? The main benefit would be a slight simplification to the CI scripts.